server/lib/flow-event.js

```diff
@@ -44,6 +44,10 @@ const UTM_PATTERN = /^[\w.%-]+$/;
 
 const IS_DISABLED = config.get('client_metrics').stderr_collector_disabled;
 
+const PERFORMANCE_PROPERTIES = new Set([
+  'responseStart', 'domInteractive', 'domComplete', 'loadEventEnd'
```

Comment:
Another performance related event that might be useful, even though it's not an official navigationTiming stat, is `loaded`. `loaded` is emit after the first view is visible, meaning it happens after all JS, CSS, HTML is downloaded (unless done using requireOnDemand) and after any XHR requests to determine the user's auth status have completed.